CHANGELOG.md

```diff
@@ -40,7 +39,7 @@
 * [ENHANCEMENT] Add runutil.CloseWithLogOnErr function. #58
 * [ENHANCEMENT] Optimise memberlist receive path when used as a backing store for rings with a large number of members. #76 #77 #84 #91 #93
 * [ENHANCEMENT] Memberlist: prepare the data to send on the write before starting counting the elapsed time for `-memberlist.packet-write-timeout`, in order to reduce chances we hit the timeout when sending a packet to other node. #89
-* [ENHANCEMENT] Memberlist: parallelize processing of messages received by memberlist. #110 
+* [ENHANCEMENT] Memberlist: parallelize processing of messages received by memberlist. #110
```

Comment:
spurious change?

Reply:
@bboreham just my editor removing trailing whitespace. Shouldn't be a problem, right?